pandas/formats/format.py

```diff
@@ -2260,6 +2260,60 @@ def _format_strings(self):
         return fmt_values
 
 
+def format_percentiles(percentiles):
+    """
+    Outputs rounded and formatted percentiles.
+
+    Parameters
+    ----------
+    percentiles : list-like, containing floats from interval [0,1]
```

Comment:
I slightly changed the design and don't require here the input to be sorted or unique. Uniqueness will be checked inside `describe`.